fdbserver/include/fdbserver/DataDistributionTeam.h

```diff
@@ -69,9 +69,41 @@ FDB_DECLARE_BOOLEAN_PARAM(ForReadBalance);
 FDB_DECLARE_BOOLEAN_PARAM(PreferLowerReadUtil);
 FDB_DECLARE_BOOLEAN_PARAM(FindTeamByServers);
 
+class TeamSelect {
+public:
+	enum Value : int8_t {
+		INVALID = 0,
+		AVOIDMOVEMENT, // Avoid data movement by selecting srcTeam if it's not busy
+		WANTSRCSERVERS, // Ask for srcTeam
+		WANTTRUEBEST, // Ask for the most or least utilized team
+		OTHER
```

Comment:
Would "ANY" or something similar be more appropriate? Other might imply that we're looking for something specific that's not on the list.

Reply:
Yes, that makes more sense.